src/classes/PMT_Payment_TEST.cls

```diff
@@ -837,6 +837,90 @@ private class PMT_Payment_TEST {
     }
 
 
+    /**
+     * @description Check that the opportunity amount is updated when the payed payments amounts
+     * exceed the opportunity amount
+     */
+    @isTest
+    private static void testOpportunityAmountSync() {
+        UTIL_CustomSettingsFacade.getContactsSettingsForTests(new npe01__Contacts_And_Orgs_Settings__c (npe01__Payments_Enabled__c = false, Enforce_Accounting_Data_Consistency__c = true));
```

Comment:
Why is Payments Enabled false?

Reply:
Does it automatically create a payment when the opportunity is created if the Payments enables is true?

Reply:
This disables the whole PMT_PaymentCreator trigger.  There is other functionality in there we would want to run in a unit test to make sure there is not breakage between our change and that code.  A better option is to set the field npe01__Do_Not_Automatically_Create_Payment__c to true on the opportunity to tell it not to create the payment automatically if needed.